tests/kitchensink/bytes.proto

```diff
@@ -6,22 +6,23 @@ import "validate/validate.proto";
 message Bytes {
     bytes none = 1;
 
-    bytes min     = 2 [(validate.rules).bytes.min_len = 3];
-    bytes max     = 3 [(validate.rules).bytes.max_len = 10];
-    bytes min_max = 4 [(validate.rules).bytes = { min_len: 3, max_len: 10 }];
-    bytes eq_len  = 5 [(validate.rules).bytes = { min_len: 5, max_len: 5}];
+	bytes len     = 2 [(validate.rules).bytes.len = 5];
```

Comment:
nit: consistent indentation

Reply:
Sorry, I'm not sure what is ‘nit’ means? My english is not good enough to understand. QAQ

Reply:
I was indicating that the comment ^ is me nitpicking - the indent fix would only be cosmetic and would not affect the actual functionality.

Reply:
Actually I did make them aligning in my editor, that may cased because of the '\t' and ' ', I'll use white space to re-ident them.